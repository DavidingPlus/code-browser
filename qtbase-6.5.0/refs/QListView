<inh f='qtbase-6.5.0/src/widgets/itemviews/qabstractitemview.h' l='25' c='QAbstractItemView'/>
<def f='qtbase-6.5.0/src/widgets/itemviews/qlistview.h' l='16' ll='165'/>
<use f='qtbase-6.5.0/src/widgets/itemviews/qlistview.h' l='164' c='_ZN9QListViewC1ERKS_'/>
<use f='qtbase-6.5.0/src/widgets/itemviews/qlistview.h' l='164' c='_ZN9QListViewaSERKS_'/>
<use f='qtbase-6.5.0/src/widgets/itemviews/qlistview.h' l='164' c='_ZN9QListViewaSERKS_'/>
<ovr f='qtbase-6.5.0/src/widgets/itemviews/qlistwidget.h' l='163' c='QListWidget'/>
<use f='qtbase-6.5.0/src/widgets/itemviews/qlistwidget.h' l='163'/>
<ovr f='qtbase-6.5.0/src/widgets/util/qundoview.h' l='21' c='QUndoView'/>
<use f='qtbase-6.5.0/src/widgets/util/qundoview.h' l='21'/>
<size>40</size>
<smbr r='QListView::staticMetaObject' t='const QMetaObject'/>
<fun r='_ZNK9QListView10metaObjectEv'/>
<fun r='_ZN9QListView11qt_metacastEPKc'/>
<fun r='_ZN9QListView11qt_metacallEN11QMetaObject4CallEiPPv'/>
<fun r='_ZN9QListView2trEPKcS1_i'/>
<fun r='_ZN9QListView18qt_static_metacallEP7QObjectN11QMetaObject4CallEiPPv'/>
<fun r='_ZN9QListViewC1EP7QWidget'/>
<fun r='_ZN9QListViewD1Ev'/>
<fun r='_ZN9QListView11setMovementENS_8MovementE'/>
<fun r='_ZNK9QListView8movementEv'/>
<fun r='_ZN9QListView7setFlowENS_4FlowE'/>
<fun r='_ZNK9QListView4flowEv'/>
<fun r='_ZN9QListView11setWrappingEb'/>
<fun r='_ZNK9QListView10isWrappingEv'/>
<fun r='_ZN9QListView13setResizeModeENS_10ResizeModeE'/>
<fun r='_ZNK9QListView10resizeModeEv'/>
<fun r='_ZN9QListView13setLayoutModeENS_10LayoutModeE'/>
<fun r='_ZNK9QListView10layoutModeEv'/>
<fun r='_ZN9QListView10setSpacingEi'/>
<fun r='_ZNK9QListView7spacingEv'/>
<fun r='_ZN9QListView12setBatchSizeEi'/>
<fun r='_ZNK9QListView9batchSizeEv'/>
<fun r='_ZN9QListView11setGridSizeERK5QSize'/>
<fun r='_ZNK9QListView8gridSizeEv'/>
<fun r='_ZN9QListView11setViewModeENS_8ViewModeE'/>
<fun r='_ZNK9QListView8viewModeEv'/>
<fun r='_ZN9QListView18clearPropertyFlagsEv'/>
<fun r='_ZNK9QListView11isRowHiddenEi'/>
<fun r='_ZN9QListView12setRowHiddenEib'/>
<fun r='_ZN9QListView14setModelColumnEi'/>
<fun r='_ZNK9QListView11modelColumnEv'/>
<fun r='_ZN9QListView19setUniformItemSizesEb'/>
<fun r='_ZNK9QListView16uniformItemSizesEv'/>
<fun r='_ZN9QListView11setWordWrapEb'/>
<fun r='_ZNK9QListView8wordWrapEv'/>
<fun r='_ZN9QListView23setSelectionRectVisibleEb'/>
<fun r='_ZNK9QListView22isSelectionRectVisibleEv'/>
<fun r='_ZN9QListView16setItemAlignmentE6QFlagsIN2Qt13AlignmentFlagEE'/>
<fun r='_ZNK9QListView13itemAlignmentEv'/>
<fun r='_ZNK9QListView10visualRectERK11QModelIndex'/>
<fun r='_ZN9QListView8scrollToERK11QModelIndexN17QAbstractItemView10ScrollHintE'/>
<fun r='_ZNK9QListView7indexAtERK6QPoint'/>
<fun r='_ZN9QListView13doItemsLayoutEv'/>
<fun r='_ZN9QListView5resetEv'/>
<fun r='_ZN9QListView12setRootIndexERK11QModelIndex'/>
<fun r='_ZN9QListView12indexesMovedERK5QListI11QModelIndexE'/>
<fun r='_ZN9QListViewC1ER16QListViewPrivateP7QWidget'/>
<fun r='_ZN9QListView5eventEP6QEvent'/>
<fun r='_ZN9QListView16scrollContentsByEii'/>
<fun r='_ZN9QListView14resizeContentsEii'/>
<fun r='_ZNK9QListView12contentsSizeEv'/>
<fun r='_ZN9QListView11dataChangedERK11QModelIndexS2_RK5QListIiE'/>
<fun r='_ZN9QListView12rowsInsertedERK11QModelIndexii'/>
<fun r='_ZN9QListView20rowsAboutToBeRemovedERK11QModelIndexii'/>
<fun r='_ZN9QListView14mouseMoveEventEP11QMouseEvent'/>
<fun r='_ZN9QListView17mouseReleaseEventEP11QMouseEvent'/>
<fun r='_ZN9QListView10wheelEventEP11QWheelEvent'/>
<fun r='_ZN9QListView10timerEventEP11QTimerEvent'/>
<fun r='_ZN9QListView11resizeEventEP12QResizeEvent'/>
<fun r='_ZN9QListView13dragMoveEventEP14QDragMoveEvent'/>
<fun r='_ZN9QListView14dragLeaveEventEP15QDragLeaveEvent'/>
<fun r='_ZN9QListView9dropEventEP10QDropEvent'/>
<fun r='_ZN9QListView9startDragE6QFlagsIN2Qt10DropActionEE'/>
<fun r='_ZNK9QListView18initViewItemOptionEP20QStyleOptionViewItem'/>
<fun r='_ZN9QListView10paintEventEP11QPaintEvent'/>
<fun r='_ZNK9QListView16horizontalOffsetEv'/>
<fun r='_ZNK9QListView14verticalOffsetEv'/>
<fun r='_ZN9QListView10moveCursorEN17QAbstractItemView12CursorActionE6QFlagsIN2Qt16KeyboardModifierEE'/>
<fun r='_ZNK9QListView12rectForIndexERK11QModelIndex'/>
<fun r='_ZN9QListView19setPositionForIndexERK6QPointRK11QModelIndex'/>
<fun r='_ZN9QListView12setSelectionERK5QRect6QFlagsIN19QItemSelectionModel13SelectionFlagEE'/>
<fun r='_ZNK9QListView24visualRegionForSelectionERK14QItemSelection'/>
<fun r='_ZNK9QListView15selectedIndexesEv'/>
<fun r='_ZN9QListView16updateGeometriesEv'/>
<fun r='_ZNK9QListView13isIndexHiddenERK11QModelIndex'/>
<fun r='_ZN9QListView16selectionChangedERK14QItemSelectionS2_'/>
<fun r='_ZN9QListView14currentChangedERK11QModelIndexS2_'/>
<fun r='_ZNK9QListView16viewportSizeHintEv'/>
<fun r='_ZNK9QListView11visualIndexERK11QModelIndex'/>
<fun r='_ZN9QListView6d_funcEv'/>
<fun r='_ZNK9QListView6d_funcEv'/>
<fun r='_ZN9QListViewC1ERKS_'/>
<use f='qtbase-6.5.0/src/widgets/accessible/itemviews.cpp' l='65' c='_ZN16QAccessibleTableC1EP7QWidget'/>
<size>40</size>
<ovr f='qtbase-6.5.0/src/widgets/dialogs/qsidebar_p.h' l='86' c='QSidebar'/>
<use f='qtbase-6.5.0/src/widgets/dialogs/qsidebar_p.h' l='86'/>
<ovr f='qtbase-6.5.0/src/widgets/dialogs/qfiledialog_p.h' l='293' c='QFileDialogListView'/>
<use f='qtbase-6.5.0/src/widgets/dialogs/qfiledialog_p.h' l='293'/>
<use f='qtbase-6.5.0/src/widgets/dialogs/qfiledialog_p.h' l='296' c='_ZN19QFileDialogListViewC1EP7QWidget'/>
<size>40</size>
<ovr f='qtbase-6.5.0/src/widgets/dialogs/qfontdialog.cpp' l='32' c='QFontListView'/>
<use f='qtbase-6.5.0/src/widgets/dialogs/qfontdialog.cpp' l='32'/>
<use f='qtbase-6.5.0/src/widgets/dialogs/qfontdialog.cpp' l='66' c='_ZN13QFontListViewC1EP7QWidget'/>
<size>40</size>
<ovr f='qtbase-6.5.0/src/widgets/dialogs/qinputdialog.cpp' l='130' c='QInputDialogListView'/>
<use f='qtbase-6.5.0/src/widgets/dialogs/qinputdialog.cpp' l='130'/>
<use f='qtbase-6.5.0/src/widgets/dialogs/qinputdialog.cpp' l='133' c='_ZN20QInputDialogListViewC1EP7QWidget'/>
<size>40</size>
<use f='qtbase-6.5.0/src/widgets/dialogs/qsidebar.cpp' l='344' c='_ZN8QSidebarC1EP7QWidget'/>
<size>40</size>
<use f='qtbase-6.5.0/src/widgets/itemviews/qcolumnview.cpp' l='709' c='_ZN11QColumnView12createColumnERK11QModelIndex'/>
<use f='qtbase-6.5.0/src/widgets/itemviews/qcolumnview.cpp' l='709' c='_ZN11QColumnView12createColumnERK11QModelIndex'/>
<size>40</size>
<use f='qtbase-6.5.0/src/widgets/itemviews/qlistview_p.h' l='84' c='_ZN19QCommonListViewBaseC1EP9QListViewP16QListViewPrivate'/>
<use f='qtbase-6.5.0/src/widgets/itemviews/qlistview_p.h' l='154'/>
<use f='qtbase-6.5.0/src/widgets/itemviews/qlistview_p.h' l='163' c='_ZN17QListModeViewBaseC1EP9QListViewP16QListViewPrivate'/>
<use f='qtbase-6.5.0/src/widgets/itemviews/qlistview_p.h' l='215' c='_ZN17QIconModeViewBaseC1EP9QListViewP16QListViewPrivate'/>
<use f='qtbase-6.5.0/src/widgets/itemviews/qlistview_p.h' l='265' c='_ZN16QListViewPrivate6q_funcEv'/>
<use f='qtbase-6.5.0/src/widgets/itemviews/qlistview_p.h' l='265' c='_ZN16QListViewPrivate6q_funcEv'/>
<use f='qtbase-6.5.0/src/widgets/itemviews/qlistview_p.h' l='265' c='_ZNK16QListViewPrivate6q_funcEv'/>
<use f='qtbase-6.5.0/src/widgets/itemviews/qlistview_p.h' l='265' c='_ZNK16QListViewPrivate6q_funcEv'/>
<use f='qtbase-6.5.0/src/widgets/itemviews/qlistview_p.h' l='265'/>
<use f='qtbase-6.5.0/src/widgets/itemviews/qlistview.cpp' l='556' c='_ZNK16QListViewPrivate23horizontalScrollToValueERK11QModelIndexRK5QRectN17QAbstractItemView10ScrollHintE'/>
<use f='qtbase-6.5.0/src/widgets/itemviews/qlistview.cpp' l='570' c='_ZNK16QListViewPrivate21verticalScrollToValueERK11QModelIndexRK5QRectN17QAbstractItemView10ScrollHintE'/>
<use f='qtbase-6.5.0/src/widgets/itemviews/qlistview.cpp' l='620' c='_ZNK16QListViewPrivate19draggablePaintPairsERK5QListI11QModelIndexEP5QRect'/>
<use f='qtbase-6.5.0/src/widgets/itemviews/qlistview.cpp' l='1783' c='_ZN16QListViewPrivate18prepareItemsLayoutEv'/>
<use f='qtbase-6.5.0/src/widgets/itemviews/qlistview.cpp' l='1854' c='_ZNK16QListViewPrivate13mapToViewportERK5QRectb'/>
<use f='qtbase-6.5.0/src/widgets/itemviews/qlistview.cpp' l='1902' c='_ZNK16QListViewPrivate8itemSizeERK20QStyleOptionViewItemRK11QModelIndex'/>
<use f='qtbase-6.5.0/src/widgets/itemviews/qlistview.cpp' l='2113' c='_ZN17QListModeViewBaseC1EP9QListViewP16QListViewPrivate'/>
<size>40</size>
<doc f='qtbase-6.5.0/src/widgets/itemviews/qlistview.cpp' l='35'>/*!
    \class QListView

    \brief The QListView class provides a list or icon view onto a model.

    \ingroup model-view
    \ingroup advanced
    \inmodule QtWidgets

    \image windows-listview.png

    A QListView presents items stored in a model, either as a simple
    non-hierarchical list, or as a collection of icons. This class is used
    to provide lists and icon views that were previously provided by the
    \c QListBox and \c QIconView classes, but using the more flexible
    approach provided by Qt&apos;s model/view architecture.

    The QListView class is one of the \l{Model/View Classes}
    and is part of Qt&apos;s \l{Model/View Programming}{model/view framework}.

    This view does not display horizontal or vertical headers; to display
    a list of items with a horizontal header, use QTreeView instead.

    QListView implements the interfaces defined by the
    QAbstractItemView class to allow it to display data provided by
    models derived from the QAbstractItemModel class.

    Items in a list view can be displayed using one of two view modes:
    In \l ListMode, the items are displayed in the form of a simple list;
    in \l IconMode, the list view takes the form of an \e{icon view} in
    which the items are displayed with icons like files in a file manager.
    By default, the list view is in \l ListMode. To change the view mode,
    use the setViewMode() function, and to determine the current view mode,
    use viewMode().

    Items in these views are laid out in the direction specified by the
    flow() of the list view. The items may be fixed in place, or allowed
    to move, depending on the view&apos;s movement() state.

    If the items in the model cannot be completely laid out in the
    direction of flow, they can be wrapped at the boundary of the view
    widget; this depends on isWrapping(). This property is useful when the
    items are being represented by an icon view.

    The resizeMode() and layoutMode() govern how and when the items are
    laid out. Items are spaced according to their spacing(), and can exist
    within a notional grid of size specified by gridSize(). The items can
    be rendered as large or small icons depending on their iconSize().

    \section1 Improving Performance

    It is possible to give the view hints about the data it is handling in order
    to improve its performance when displaying large numbers of items. One approach
    that can be taken for views that are intended to display items with equal sizes
    is to set the \l uniformItemSizes property to true.

    \sa {View Classes}, {Item Views Puzzle Example}, QTreeView, QTableView, QListWidget
*/</doc>
<fun r='_ZN9QListViewC1EP7QWidget'/>
<fun r='_ZN9QListViewC1ER16QListViewPrivateP7QWidget'/>
<fun r='_ZN9QListViewD1Ev'/>
<fun r='_ZN9QListView11setMovementENS_8MovementE'/>
<fun r='_ZNK9QListView8movementEv'/>
<fun r='_ZN9QListView7setFlowENS_4FlowE'/>
<fun r='_ZNK9QListView4flowEv'/>
<fun r='_ZN9QListView11setWrappingEb'/>
<fun r='_ZNK9QListView10isWrappingEv'/>
<fun r='_ZN9QListView13setResizeModeENS_10ResizeModeE'/>
<fun r='_ZNK9QListView10resizeModeEv'/>
<fun r='_ZN9QListView13setLayoutModeENS_10LayoutModeE'/>
<fun r='_ZNK9QListView10layoutModeEv'/>
<fun r='_ZN9QListView10setSpacingEi'/>
<fun r='_ZNK9QListView7spacingEv'/>
<fun r='_ZN9QListView12setBatchSizeEi'/>
<fun r='_ZNK9QListView9batchSizeEv'/>
<fun r='_ZN9QListView11setGridSizeERK5QSize'/>
<fun r='_ZNK9QListView8gridSizeEv'/>
<fun r='_ZN9QListView11setViewModeENS_8ViewModeE'/>
<fun r='_ZNK9QListView8viewModeEv'/>
<fun r='_ZN9QListView18clearPropertyFlagsEv'/>
<fun r='_ZNK9QListView11isRowHiddenEi'/>
<fun r='_ZN9QListView12setRowHiddenEib'/>
<fun r='_ZNK9QListView10visualRectERK11QModelIndex'/>
<fun r='_ZN9QListView8scrollToERK11QModelIndexN17QAbstractItemView10ScrollHintE'/>
<fun r='_ZN9QListView5resetEv'/>
<fun r='_ZN9QListView12setRootIndexERK11QModelIndex'/>
<fun r='_ZN9QListView16scrollContentsByEii'/>
<fun r='_ZN9QListView14resizeContentsEii'/>
<fun r='_ZNK9QListView12contentsSizeEv'/>
<fun r='_ZN9QListView11dataChangedERK11QModelIndexS2_RK5QListIiE'/>
<fun r='_ZN9QListView12rowsInsertedERK11QModelIndexii'/>
<fun r='_ZN9QListView20rowsAboutToBeRemovedERK11QModelIndexii'/>
<fun r='_ZN9QListView14mouseMoveEventEP11QMouseEvent'/>
<fun r='_ZN9QListView17mouseReleaseEventEP11QMouseEvent'/>
<fun r='_ZN9QListView10wheelEventEP11QWheelEvent'/>
<fun r='_ZN9QListView10timerEventEP11QTimerEvent'/>
<fun r='_ZN9QListView11resizeEventEP12QResizeEvent'/>
<fun r='_ZN9QListView13dragMoveEventEP14QDragMoveEvent'/>
<fun r='_ZN9QListView14dragLeaveEventEP15QDragLeaveEvent'/>
<fun r='_ZN9QListView9dropEventEP10QDropEvent'/>
<fun r='_ZN9QListView9startDragE6QFlagsIN2Qt10DropActionEE'/>
<fun r='_ZNK9QListView18initViewItemOptionEP20QStyleOptionViewItem'/>
<fun r='_ZN9QListView10paintEventEP11QPaintEvent'/>
<fun r='_ZNK9QListView7indexAtERK6QPoint'/>
<fun r='_ZNK9QListView16horizontalOffsetEv'/>
<fun r='_ZNK9QListView14verticalOffsetEv'/>
<fun r='_ZN9QListView10moveCursorEN17QAbstractItemView12CursorActionE6QFlagsIN2Qt16KeyboardModifierEE'/>
<fun r='_ZNK9QListView12rectForIndexERK11QModelIndex'/>
<fun r='_ZN9QListView19setPositionForIndexERK6QPointRK11QModelIndex'/>
<fun r='_ZN9QListView12setSelectionERK5QRect6QFlagsIN19QItemSelectionModel13SelectionFlagEE'/>
<fun r='_ZNK9QListView24visualRegionForSelectionERK14QItemSelection'/>
<fun r='_ZNK9QListView15selectedIndexesEv'/>
<fun r='_ZN9QListView13doItemsLayoutEv'/>
<fun r='_ZN9QListView16updateGeometriesEv'/>
<fun r='_ZNK9QListView13isIndexHiddenERK11QModelIndex'/>
<fun r='_ZN9QListView14setModelColumnEi'/>
<fun r='_ZNK9QListView11modelColumnEv'/>
<fun r='_ZN9QListView19setUniformItemSizesEb'/>
<fun r='_ZNK9QListView16uniformItemSizesEv'/>
<fun r='_ZN9QListView11setWordWrapEb'/>
<fun r='_ZNK9QListView8wordWrapEv'/>
<fun r='_ZN9QListView23setSelectionRectVisibleEb'/>
<fun r='_ZNK9QListView22isSelectionRectVisibleEv'/>
<fun r='_ZN9QListView16setItemAlignmentE6QFlagsIN2Qt13AlignmentFlagEE'/>
<fun r='_ZNK9QListView13itemAlignmentEv'/>
<fun r='_ZN9QListView5eventEP6QEvent'/>
<fun r='_ZN9QListView14currentChangedERK11QModelIndexS2_'/>
<fun r='_ZN9QListView16selectionChangedERK14QItemSelectionS2_'/>
<fun r='_ZNK9QListView11visualIndexERK11QModelIndex'/>
<fun r='_ZNK9QListView16viewportSizeHintEv'/>
<use f='qtbase-6.5.0/src/widgets/itemviews/qlistwidget.cpp' l='1398' c='_ZN11QListWidgetC1EP7QWidget'/>
<size>40</size>
<use f='qtbase-6.5.0/src/widgets/styles/qpixmapstyle.cpp' l='251' c='_ZNK12QPixmapStyle13drawPrimitiveEN6QStyle16PrimitiveElementEPK12QStyleOptionP8QPainterPK7QWidget'/>
<size>40</size>
<use f='qtbase-6.5.0/src/widgets/styles/qwindowsstyle.cpp' l='525' c='_ZNK13QWindowsStyle9styleHintEN6QStyle9StyleHintEPK12QStyleOptionPK7QWidgetP16QStyleHintReturn'/>
<size>40</size>
<use f='qtbase-6.5.0/src/widgets/util/qcompleter.cpp' l='1241' c='_ZN10QCompleter8setPopupEP17QAbstractItemView'/>
<use f='qtbase-6.5.0/src/widgets/util/qcompleter.cpp' l='1241' c='_ZN10QCompleter8setPopupEP17QAbstractItemView'/>
<use f='qtbase-6.5.0/src/widgets/util/qcompleter.cpp' l='1266' c='_ZNK10QCompleter5popupEv'/>
<use f='qtbase-6.5.0/src/widgets/util/qcompleter.cpp' l='1266' c='_ZNK10QCompleter5popupEv'/>
<use f='qtbase-6.5.0/src/widgets/util/qcompleter.cpp' l='1608' c='_ZN10QCompleter19setCompletionColumnEi'/>
<use f='qtbase-6.5.0/src/widgets/util/qcompleter.cpp' l='1608' c='_ZN10QCompleter19setCompletionColumnEi'/>
<size>40</size>
<use f='qtbase-6.5.0/src/widgets/util/qundoview.cpp' l='267' c='_ZN9QUndoViewC1EP7QWidget'/>
<use f='qtbase-6.5.0/src/widgets/util/qundoview.cpp' l='278' c='_ZN9QUndoViewC1EP10QUndoStackP7QWidget'/>
<use f='qtbase-6.5.0/src/widgets/util/qundoview.cpp' l='294' c='_ZN9QUndoViewC1EP10QUndoGroupP7QWidget'/>
<size>40</size>
<ovr f='qtbase-6.5.0/src/widgets/widgets/qcombobox_p.h' l='48' c='QComboBoxListView'/>
<use f='qtbase-6.5.0/src/widgets/widgets/qcombobox_p.h' l='48'/>
<use f='qtbase-6.5.0/src/widgets/widgets/qcombobox.cpp' l='637' c='_ZNK25QComboBoxPrivateContainer9topMarginEv'/>
<use f='qtbase-6.5.0/src/widgets/widgets/qcombobox.cpp' l='637' c='_ZNK25QComboBoxPrivateContainer9topMarginEv'/>
<use f='qtbase-6.5.0/src/widgets/widgets/qcombobox.cpp' l='651' c='_ZNK25QComboBoxPrivateContainer7spacingEv'/>
<use f='qtbase-6.5.0/src/widgets/widgets/qcombobox.cpp' l='651' c='_ZNK25QComboBoxPrivateContainer7spacingEv'/>
<use f='qtbase-6.5.0/src/widgets/widgets/qcombobox.cpp' l='3470' c='_ZN9QComboBox14setModelColumnEi'/>
<use f='qtbase-6.5.0/src/widgets/widgets/qcombobox.cpp' l='3470' c='_ZN9QComboBox14setModelColumnEi'/>
<size>40</size>
<use f='qtbase-6.5.0/src/widgets/widgets/qfontcombobox.cpp' l='400' c='_ZN13QFontComboBoxC1EP7QWidget'/>
<use f='qtbase-6.5.0/src/widgets/widgets/qfontcombobox.cpp' l='400' c='_ZN13QFontComboBoxC1EP7QWidget'/>
<use f='qtbase-6.5.0/src/widgets/widgets/qfontcombobox.cpp' l='520' c='_ZN13QFontComboBox5eventEP6QEvent'/>
<use f='qtbase-6.5.0/src/widgets/widgets/qfontcombobox.cpp' l='520' c='_ZN13QFontComboBox5eventEP6QEvent'/>
<size>40</size>
