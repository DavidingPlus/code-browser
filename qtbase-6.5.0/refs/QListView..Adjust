<dec f='qtbase-6.5.0/src/widgets/itemviews/qlistview.h' l='39' type='1'/>
<use f='qtbase-6.5.0/src/widgets/dialogs/qfiledialog.cpp' l='4064' u='r' c='_ZN19QFileDialogListView20setFileDialogPrivateEP18QFileDialogPrivate'/>
<use f='qtbase-6.5.0/src/widgets/itemviews/qlistview.cpp' l='844' u='r' c='_ZN9QListView11resizeEventEP12QResizeEvent'/>
<doc f='qtbase-6.5.0/src/widgets/itemviews/qlistview.cpp' l='123'>\value Adjust The items will be laid out every time the view is resized.</doc>
