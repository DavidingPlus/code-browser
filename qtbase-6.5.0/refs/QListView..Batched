<dec f='qtbase-6.5.0/src/widgets/itemviews/qlistview.h' l='41' type='1'/>
<doc f='qtbase-6.5.0/src/widgets/itemviews/qlistview.cpp' l='130'>\value Batched The items are laid out in batches of \l batchSize items.
  </doc>
