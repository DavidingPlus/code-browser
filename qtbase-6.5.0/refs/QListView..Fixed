<dec f='qtbase-6.5.0/src/widgets/itemviews/qlistview.h' l='39' type='0'/>
<use f='qtbase-6.5.0/src/widgets/itemviews/qlistview.cpp' l='443' u='r' c='_ZN9QListView11setViewModeENS_8ViewModeE'/>
<use f='qtbase-6.5.0/src/widgets/itemviews/qlistview.cpp' l='459' u='r' c='_ZN9QListView11setViewModeENS_8ViewModeE'/>
<use f='qtbase-6.5.0/src/widgets/itemviews/qlistview.cpp' l='1757' u='r' c='_ZN16QListViewPrivateC1Ev'/>
<doc f='qtbase-6.5.0/src/widgets/itemviews/qlistview.cpp' l='122'>\value Fixed The items will only be laid out the first time the view is shown.
  </doc>
