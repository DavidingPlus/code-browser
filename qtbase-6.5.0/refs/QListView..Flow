<def f='qtbase-6.5.0/src/widgets/itemviews/qlistview.h' l='37'/>
<use f='qtbase-6.5.0/src/widgets/itemviews/qlistview.h' l='38' c='_Z20qt_getEnumMetaObjectN9QListView4FlowE'/>
<use f='qtbase-6.5.0/src/widgets/itemviews/qlistview.h' l='38' c='_Z14qt_getEnumNameN9QListView4FlowE'/>
<use f='qtbase-6.5.0/src/widgets/itemviews/qlistview.h' l='52' c='_ZN9QListView7setFlowENS_4FlowE'/>
<use f='qtbase-6.5.0/src/widgets/itemviews/qlistview.h' l='53' c='_ZNK9QListView4flowEv'/>
<use f='qtbase-6.5.0/src/widgets/itemviews/qlistview_p.h' l='73'/>
<use f='qtbase-6.5.0/src/widgets/itemviews/qlistview_p.h' l='123' c='_ZNK19QCommonListViewBase4flowEv'/>
<use f='qtbase-6.5.0/src/widgets/itemviews/qlistview_p.h' l='389'/>
<use f='qtbase-6.5.0/src/widgets/itemviews/qlistview_p.h' l='433' c='_ZNK19QCommonListViewBase4flowEv'/>
<use f='qtbase-6.5.0/src/widgets/itemviews/qlistview.cpp' l='231' c='_ZN9QListView7setFlowENS_4FlowE'/>
<use f='qtbase-6.5.0/src/widgets/itemviews/qlistview.cpp' l='239' c='_ZNK9QListView4flowEv'/>
<doc f='qtbase-6.5.0/src/widgets/itemviews/qlistview.cpp' l='110'>/*!
  \enum QListView::Flow

  \value LeftToRight The items are laid out in the view from the left
  to the right.
  \value TopToBottom The items are laid out in the view from the top
  to the bottom.
*/</doc>
