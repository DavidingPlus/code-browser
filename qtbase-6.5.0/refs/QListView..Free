<dec f='qtbase-6.5.0/src/widgets/itemviews/qlistview.h' l='35' type='1'/>
<use f='qtbase-6.5.0/src/widgets/itemviews/qlistview.cpp' l='457' u='r' c='_ZN9QListView11setViewModeENS_8ViewModeE'/>
<doc f='qtbase-6.5.0/src/widgets/itemviews/qlistview.cpp' l='105'>\value Free The items can be moved freely by the user.
  </doc>
