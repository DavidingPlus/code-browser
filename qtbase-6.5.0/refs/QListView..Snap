<dec f='qtbase-6.5.0/src/widgets/itemviews/qlistview.h' l='35' type='2'/>
<use f='qtbase-6.5.0/src/widgets/itemviews/qlistview.cpp' l='2933' u='r' c='_ZN17QIconModeViewBase15filterDropEventEP10QDropEvent'/>
<use f='qtbase-6.5.0/src/widgets/itemviews/qlistview.cpp' l='2990' u='r' c='_ZN17QIconModeViewBase19filterDragMoveEventEP14QDragMoveEvent'/>
<use f='qtbase-6.5.0/src/widgets/itemviews/qlistview.cpp' l='3352' u='r' c='_ZNK17QIconModeViewBase17draggedItemsDeltaEv'/>
<doc f='qtbase-6.5.0/src/widgets/itemviews/qlistview.cpp' l='106'>\value Snap The items snap to the specified grid when moved; see
  setGridSize().</doc>
